docs/faq/proof-of-stake/stake-pools.md

```diff
@@ -2,10 +2,14 @@
 
 ---
 
+!!! danger "Critical Information"
+
+    This FAQ dates from earlier in Decred's history and used deprecated terms like PoS Mining (now Voting) and Stakepools (now Voting Service Providers).
```

Comment:
Again, can we remove this disclaimer please and instead open a new issue to address these outdated terms.

Reply:
Removed disclaimer. Created Issue: https://github.com/decred/dcrdocs/issues/713